swift/ITSClient/Sources/ITSMobility/Mobility.swift

```diff
@@ -45,6 +45,33 @@ public actor Mobility {
         }
     }
 
+    /// Starts the `Mobility` mainly with a bootstrap to connect to a MQTT server and initialize the telemetry client.
+    /// - Parameters:
+    ///   - bootstrap: The `Bootstrap` which can be retrieved with `BootstrapService`.
+    ///   - stationID: The station identifier. Must be unique and the same each time on a same device.
```

Comment:
> `the same each time on a same device`

That's actually not true. A station may decide to cycle its `stationID` to thwart re-identification, for example.